ValidationLibrary.AzureFunctions/StatusEndpoint.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace ValidationLibrary.AzureFunctions
+{
+    public class StatusEndpoint
+    {
+        private readonly ILogger<StatusEndpoint> _logger;
+        private ValidationLibrary.RepositoryValidator _validator;
+        public StatusEndpoint(ILogger<StatusEndpoint> logger, ValidationLibrary.RepositoryValidator validator)
+        {
+            _logger = logger;
+            _validator = validator;
+        }
+
+        [FunctionName("StatusCheck")]
+        public IActionResult Run([HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequestMessage req)
+        {
+            try
+            {
+                _logger.LogDebug("Repository validator status check hook launched.");
+
+                var status = new Status
+                {
+                    readmeTemplateFileLocation = "https://raw.githubusercontent.com/protacon/repository-validator/master/README_TEMPLATE.md",
```

Comment:
Don't use this here, we want to return it from `HasReadmeRule`

Reply:
Oh yes of course